src/lantmateriet/extract.py

```diff
@@ -0,0 +1,105 @@
+"""Extract GeoJSON from GPKG files."""
+
+import glob
+import logging
+from pathlib import Path
+from typing import Union
+
+import fiona
+import pandas as pd
+import shapely
+from lantmateriet.config import Config50, config_50
+from lantmateriet.line import Line
+from lantmateriet.point import Point
+from lantmateriet.polygon import Polygon
+from lantmateriet.utils import normalise_item_names, read_first_entry, read_unique_names
+from ray.util.multiprocessing import Pool
+
+file_geometry_mapping: dict[shapely.Geometry, Union[Line, Polygon, Point]] = {
```

Comment:
Not sure as I'd like to avoid importing `Line`, `Polygon` and `Point` in `config`. I think we can remove or simplify `Line`, `Polygon` and `Point` classes altogether and put them in the geometry file.

Reply:
sounds good.

Reply:
I'll create an issue about this.